microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md

```diff
@@ -89,6 +89,28 @@ You can define mail flow rules for triggering message encryption with the new OM
 
 8. Choose **Save**.
 
+## Create mail flow rules to remove encryption for outgoing email messages with the new OME capabilities
+
+You can define mail flow rules for triggering remove message encryption with the new OME capabilities by using the EAC.
```

Comment:
You can create a mail flow rule to remove message encryption with the new OME capabilities by using the EAC.